sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -127,6 +127,9 @@ public ClientStatusResponse getClientStatus() {
                     inProgressSteps.stream().map(step -> step.getMessage()).collect(Collectors.toList()));
         }
         activeWorkSet.addAll(inProgressSteps);
+        if (workSetTracker == null) {
```

Comment:
trying to nail down why the scheduler hit a null ref outlined here https://jira.mesosphere.com/browse/DCOS-40696, workset tracker is initialized in the registered call ( public void registered(boolean reRegistered on line 107) the theory is that the scheduler hit this block without initializing workset tracker, if we see this log line it will indicate that this theory is true